### PR TITLE
add CoLinksSupplyEligibility

### PR DESCRIPTION
**Pull request by spengrah**
This PR submits a new module to the registry: [CoLinksEligibility](https://github.com/Hats-Protocol/colinks-eligibility)

This module determines eligibility based on the (configurable) supply of the potential wearer's [CoLinks](https://colinks.coordinape.com/) links. The module is very simple, with new mutable parameters or extra functions beyond the standard `IHatsEligibility.getWearerStatus()`.

If added to the registry...

- Users of Hats apps will be able to leverage the fuzz-yet-credible signal of reputation reflected by an accounts link supply.
- The CoLinks community will be able to use Hats to expand the set of tools they can access.

### CODE REVIEW

**Review by gershido — COMMENTED** [review]
Looks good!  two small notes:
1. Regarding the copy here:
https://github.com/Hats-Protocol/modules-registry/blob/cdc9d2649fce88a685a58c019e05b883fde2a035/modules/coLinksEligibility.json#L4
For me it will be clearer to explicitly say that "supply" means the amount of links to the individual that were minted (can be confused with the the amount of links to others that the individual holds).
2. Need to run the formatter.

**Review by gershido — APPROVED** [review]
Lgtm!

File: modules/coLinksSupplyEligibility.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      "chainId": "10",
+      "block": "114023601"
+    }
+  ],
+  "creationArgs": {
+    "useHatId": true,
+    "immutable": [
+      {
+        "name": "Link Supply Threshold",
+        "description": "The minimum supply of an account's links that must for the account to be eligible.",
```
Comment (nintynick) [inline modules/coLinksSupplyEligibility.json:46]: typo

**Review by nintynick — APPROVED** [review]
lgtm

**Pull request merged**